### DISCUSSION TITLE
Problem in map reload/refresh

### DISCUSSION THREAD

**Discussion by ShaneChen0122** [changdoc/naver-vaccine-macro#142] [Q&A]
Hello author,
Thank you for your plugin, it is very useful for me.
however, I met a problem when I use refresh in naver map page.
Firstly, It is ok when I started to click the button, and the program executed well.
But, around 20min past, I can not request any update date from sever.
So, I check the error info, it shows the sever refused my request.
I guess it is too fast to ask request from sever.
I try to change the value (random number from 2000 to 5000)of RELOAD_INTERVAL_MILLISECONDS in map.js, it looks problem solved. 
The weakness of solution is  reduce the probability of catch the vaccine.
It is just a tip for solved problem who met same problem with me.
Of course, hope author have more great way to solve.

By the way, my system version is macOS Big Sur 11.0.1, Chorme vesion is  92.0.4515.107

**Comment by changdoc** [comment]
Hello ShaneChen0122
Did you mean below api call(graphql) that your request refused from server?
![image](https://user-images.githubusercontent.com/1879494/127319131-07e75dd7-dec9-4b53-9abc-bc1d8cd0286f.png)
I coudn't get error from server while refresh in map. Give me more info about error and situations.

Anyway Refresh in map is under developing, not stable.
So if you need booking for vaccine with naver, launch macro on reservation page. (ex: https://v-search.nid.naver.com/reservation?orgCd=XXX&sid=XXXX) guide in #4
